assertsprocessor/metric_helper.go

```diff
@@ -174,6 +172,7 @@ func (p *metricHelper) buildLabels(namespace string, service string, span *ptrac
 		}
 	}
 	labels[spanKind] = span.Kind().String()
+	labels[statusCode] = span.Status().Code().String()
```

Comment:
Is the span status code always set? Could `.Code()` or `.String()` throw a nil deference error?

Reply:
It returns `Unset` as the status code if there is no status code in the trace. I've updated a test case to cover this scenario